js/kucoinfutures.js

```diff
@@ -1188,12 +1194,15 @@ module.exports = class kucoinfutures extends kucoin {
         const id = this.safeString (info, 'id');
         market = this.safeMarket (id, market);
         const currencyId = this.safeString (info, 'settleCurrency');
+        const mode = (this.safeValue (info, 'crossMode') === true) ? 'cross' : 'isolated';
+        const marketId = this.safeString (market, 'symbol');
         return {
             'info': info,
             'type': undefined,
+            'mode': mode,
             'amount': undefined,
             'code': this.safeCurrencyCode (currencyId),
-            'symbol': this.safeString (market, 'symbol'),
+            'symbol': this.safeSymbol (marketId, market),
```

Comment:
safeSymbol here does absolutely nothing, and the `marketId` is actually the symbol, because it was taken from the market. safeSymbol would just be trying to take the symbol from the market a second time

Reply:
yes as you see, that's old PR, and yesterday I've commented under your PR, that `market` variable was mistake there, should have been `info`, from where we get raw symbol.